backend/inspirehep/records/serializers/templates/latex_template.tex

```diff
@@ -9,14 +9,15 @@
 \VAR{- '\n' if 'authors' in data or 'collaborations' in data }
 {%- if 'collaborations' in data -%}
     {%- if 'authors' in data -%}
-        \VAR{ data.authors.0 }
-        \VAR{- ' \\textit{et al.}' if data.authors|length > 1 }
+        \VAR{ data.authors[:10] ~ ',' }
+        \VAR{- ' \\textit{et al.}' if data.authors|length > 10 }
```

Comment:
this can stay like it was, it's the non-collaboration case that should be fixed.

Reply:
Ok I'll change it, in the refinement it was written to fix both